Data/sync/SyncCrypto.swift

```diff
@@ -38,10 +38,9 @@ class SyncCrypto: JSInjector {
         return webCfg
     }
     
-    ///
-    func uniqueSerialBytes(count byteCount: Int) -> [Int]? {
+    class func uniqueSerialBytes(count byteCount: Int) -> [Int]? {
         if byteCount % 2 != 0 { return nil }
-        return (0..<byteCount).map { _ in Int(arc4random_uniform(256)) }
+        return (0..<byteCount).map { _ in Int.random(in: 0 ... 256) }
```

Comment:
This is iOS 12+ only? Also, this is inclusive (`...`) so will give 257 number options. Too many, should either be `0 ..< 256` or `0 ... 255`.

Reply:
I will revert to the old implementation, `Int.random` is available in Swift 4.2